app/src/main/res/layout/action_bar.xml

```diff
@@ -47,7 +47,7 @@
           android:layout_marginLeft="26dp"
           android:layout_marginStart="26dp"
           android:layout_marginTop="12dp"
-          android:background="@drawable/updates_badge_circle"
+          android:background="@drawable/updates_badge_circle_border"
```

Comment:
It looks that we were using the updates badge for the notification dot in the promotions. Since it is a new file now, as it should, we should change the name to something that makes sense (this is not updates)